### PR TITLE
CATROID-1291 Select which object from which scene from another local project to add to the project's current scene

### PR DESCRIPTION
**Pull request by mseiser**
When importing from local project, users can now choose
- a scene from which to choose an object, if it's a multiple scene project
- a specific object of that scene or the default scene

New Tests
- ImportLocalActivityTest
- ImportLocalSpriteToProjectTest.importLocalSpriteFromMultipleSceneProject

Renamed/Changed Tests 
- ImportLocalSpriteTest -> ImportLocalSpriteToProjectTest 
- MergeLocalSpriteTest -> ImportLocalSpriteToSpriteTest
- ImportObjectIntoProjectFromContextMenuTest -> ImportMediaObjectToSpriteTest
- ImportObjectIntoProjectTest -> ImportMediaObjectToProjectTest

https://jira.catrob.at/browse/CATROID-1291

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: catroid/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -504,6 +504,7 @@
     <string name="no_projects_selected">沒有選擇要導入的項目</string>
     <string name="only_select_catrobat_files">只選擇項目文件</string>
     <string name="error_no_files_found">未找到項目文件。</string>
+    <!--suppress UnusedResources -->
```
Comment (bengaal) [inline catroid/src/main/res/values-zh-rTW/strings.xml:507]: Deleting the not used resource could be better than suppressing the warning

File: catroid/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1422,6 +1427,8 @@ needs read and write access to it. You can always change permissions through you
         started, seems there is a problem with our server."</string>
     <string name="error_new_project">Something went wrong while creating a new project.</string>
     <string name="error_rename_scene">Something went wrong while renaming the scene.</string>
+    <!--suppress UnusedResources -->
```
Comment (bengaal) [inline catroid/src/main/res/values/strings.xml:1430]: Deleting the not used resource could be better than suppressing the warning

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -70,7 +70,6 @@
       <option name="XML_KEEP_WHITESPACES" value="true" />
       <option name="XML_ATTRIBUTE_WRAP" value="0" />
       <option name="XML_TEXT_WRAP" value="0" />
-      <option name="XML_LEGACY_SETTINGS_IMPORTED" value="true" />
```
Comment (bengaal) [inline .idea/codeStyles/Project.xml:73]: Why did we change this?

**Review by bengaal — APPROVED** [review]
Few things to fine tune, but looks good to me in general. We reviewed it in PP.

**Comment by mseiser** [comment]
This is replaced by https://github.com/Catrobat/Catroid/pull/4635

**Pull request closed without merging**